frontend/catalyst/compilation_pipelines.py

```diff
@@ -812,16 +812,18 @@ def qjit(
 
     .. note::
 
-        Currently, ``lightning.qubit`` is the only supported backend device
-        for Catalyst compilation. For a list of supported operations, observables,
+        The supported backend devices are ``lightning.qubit``, ``lightning.kokkos``,
+        ``braket.local.qubit``, and ``braket.aws.qubit``.
+        For a list of supported operations, observables,
```

Comment:
@maliasadi @dime10 this is the only change in this file not previously reviewed.